test/models/comment_test.rb

```diff
@@ -1,7 +1,24 @@
-require 'test_helper'
+require 'rails_helper'
```

Comment:
- [OPTIONAL] This was not mentioned by the previous reviewer, I noticed an issue with the recent migration. The error indicates that the "posts" table is not found during the execution of the "CreateComments" migration.

This typically happens when the order of migrations is not aligned with the dependencies between tables. Since "comments" likely has a foreign key reference to "posts," the "posts" table should be created before "comments."

To address this, you can try either of the following:

Regenerate Migrations:
1. Delete the existing migration files and generate them again, ensuring that "CreatePosts" comes before "CreateComments." 

```ruby
# Example commands
rm db/migrate/*_create_comments.rb
rails generate migration CreatePosts
rails generate migration CreateComments
```
2. Manual Adjustment:
If you prefer, you can manually adjust the timestamp in the migration file names to ensure the correct order.
```ruby
# Example filenames in db/migrate/
20231117004156_create_posts.rb
20231117210241_create_comments.rb
```

- - Screenshot reference:
![Selection_039](https://github.com/CarlosZ96/Blog-App-MCV/assets/107632780/10d2ca64-401c-4685-9563-cb8d94b9c2b3)